### ISSUE TITLE
open62541 build broken

### ISSUE THREAD

**Issue by inferno-chromium** [google/oss-fuzz#1192]
https://oss-fuzz-build-logs.storage.googleapis.com/index.html

@Pro - can you please take a look. I did an attempt with https://github.com/google/oss-fuzz/commit/90f98631d8badf4bf6a934823ec0b4db72772744, but looks like there are more dependencies.

**Comment by Pro** [comment]
You probably forgot to also set the version for `libmbedcrypto0_2.6.0-1_amd64.deb`.
Can you try that?

I can not see the old build error, but there was a general build error on master, which was fixed by the commit:
https://github.com/open62541/open62541/pull/1614

**Comment by inferno-chromium** [comment]
@Pro - thanks for spotting that which fixed that, now it is failing here on some deprecated calls.

```
I  ERROR: build step 4 "gcr.io/oss-fuzz/open62541" failed: exit status 2 
I  ERROR 
I  Finished Step #4 
I  Step #4: make: *** [all] Error 2 
I  Step #4: Makefile:149: recipe for target 'all' failed 
I  Step #4: CMakeFiles/Makefile2:135: recipe for target 'CMakeFiles/open62541-plugins.dir/all' failed 
I  Step #4: make[1]: *** [CMakeFiles/open62541-plugins.dir/all] Error 2 
I  Step #4: make[2]: *** [CMakeFiles/open62541-plugins.dir/plugins/ua_securitypolicy_basic128rsa15.c.o] Error 1 
I  Step #4: CMakeFiles/open62541-plugins.dir/build.make:270: recipe for target 'CMakeFiles/open62541-plugins.dir/plugins/ua_securitypolicy_basic128rsa15.c.o' failed 
I  Step #4: 3 errors generated. 
I  Step #4:                                                ^ 
I  Step #4: #define MBEDTLS_DEPRECATED      __attribute__((deprecated)) 
I  Step #4: /usr/include/mbedtls/sha1.h:302:48: note: expanded from macro 'MBEDTLS_DEPRECATED' 
I  Step #4: ^ 
I  Step #4: MBEDTLS_DEPRECATED static inline void mbedtls_sha1( const unsigned char *input, 
I  Step #4: /usr/include/mbedtls/sha1.h:320:1: note: 'mbedtls_sha1' has been explicitly marked deprecated here 
I  Step #4:     ^ 
I  Step #4:     mbedtls_sha1(certificate->data, certificate->length, thumbprint->data); 
I  Step #4: /src/open62541/plugins/ua_securitypolicy_basic128rsa15.c:265:5: error: 'mbedtls_sha1' is deprecated [-Werror,-Wdeprecated-declarations] 
I  Step #4:                                                ^ 
I  Step #4: #define MBEDTLS_DEPRECATED      __attribute__((deprecated)) 
I  Step #4: /usr/include/mbedtls/sha1.h:302:48: note: expanded from macro 'MBEDTLS_DEPRECATED' 
I  Step #4: ^ 
I  Step #4: MBEDTLS_DEPRECATED static inline void mbedtls_sha1( const unsigned char *input, 
I  Step #4: /usr/include/mbedtls/sha1.h:320:1: note: 'mbedtls_sha1' has been explicitly marked deprecated here 
I  Step #4:     ^ 
I  Step #4:     mbedtls_sha1(message->data, message->length, hash); 
I  Step #4: /src/open62541/plugins/ua_securitypolicy_basic128rsa15.c:113:5: error: 'mbedtls_sha1' is deprecated [-Werror,-Wdeprecated-declarations] 
I  Step #4:                                                ^ 
I  Step #4: #define MBEDTLS_DEPRECATED      __attribute__((deprecated)) 
I  Step #4: /usr/include/mbedtls/sha1.h:302:48: note: expanded from macro 'MBEDTLS_DEPRECATED' 
I  Step #4: ^ 
I  Step #4: MBEDTLS_DEPRECATED static inline void mbedtls_sha1( const unsigned char *input, 
I  Step #4: /usr/include/mbedtls/sha1.h:320:1: note: 'mbedtls_sha1' has been explicitly marked deprecated here 
I  Step #4:     ^ 
I  Step #4:     mbedtls_sha1(message->data, message->length, hash); 
I  Step #4: /src/open62541/plugins/ua_securitypolicy_basic128rsa15.c:90:5: error: 'mbedtls_sha1' is deprecated [-Werror,-Wdeprecated-declarations] 
I  Step #4: [ 14%] Building C object CMakeFiles/open62541-plugins.dir/plugins/ua_securitypolicy_basic128rsa15.c.o 
```

**Comment by Pro** [comment]
can you try to use version 2.6.0 again? We need to fix that deprecation later (see https://github.com/open62541/open62541/issues/1616)

**Comment by Pro** [comment]
You probably forgot to also set the version for `libmbedcrypto0_2.6.0-1_amd64.deb`.
Can you try that?

I can not see the old build error, but there was a general build error on master, which was fixed by the commit:
https://github.com/open62541/open62541/pull/1614

**Comment by inferno-chromium** [comment]
@Pro - the older version 2.6 of libraries like libmbedtls-dev does not exist on http://ftp.us.debian.org/debian/pool/main/m/mbedtls/, that was the reason to bump up versions. Can you bring those back and then send a pull request here to revert this. Please verify that build works using.
python infra/helper.py build_fuzzers open62541

**Comment by inferno-chromium** [comment]
@Pro - basically, looks like we need to use 2.6 versions of everything but those don't exist in http://ftp.us.debian.org/debian/pool/main/m/mbedtls/. Can you fix that.

```
dpkg: dependency problems prevent configuration of libmbedtls10:amd64:
 libmbedtls10:amd64 depends on libmbedcrypto1 (>= 2.7); however:
  Package libmbedcrypto1 is not installed.
```

**Comment by Pro** [comment]
Fixed in #1196 (tested locally and it builds)
